src/openmrs-react-root-decorator.js

```diff
@@ -23,14 +31,17 @@ export default function decorateOptions(opts) {
           // TO-DO have a UX designed for when a catastrophic error occurs
           return null;
         } else {
+          const rootComponent = (
+            <ModuleNameContext.Provider value={opts.moduleName}>
```

Comment:
`ModuleNameContext` is undefined here. I think react-root-decorator is working with a different React than esm-module-config (build-time vs run-time).

Reply:
In the browser

Reply:
I don't see how a different version of React would be happening since React is a peer dependency here. I also don't see how a different version of React would cause ModuleNameContext to be undefined.

Here are the steps I would take to debug:

1. Ensure that module-config is actually loadable
```js
// In browser console
System.import('@openmrs/esm-module-config').then(m => {
  console.log(m)
  console.log(m.ModuleNameContext)
})
```

2. Ensure that the esm repo you're using (probably esm-login, right?) has esm-module-config as a webpack external. You can verify that module-config is an external by going to the url for esm-login and verifying that it has `System.register(['@openmrs/esm-module-config'], ...)` at the beginning of it.